src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -1153,8 +1153,9 @@ class OPENASSETIO_CORE_EXPORT Manager {
    * @param entityReferences The entity references to preflight prior
    * to registration.
    *
-   * @param traitSet The @ref trait_set of the entities that are being
-   * published.
+   * @param entityTraitsDatas @ref trait_set for each entity,
+   * determining the type of entity to publish, complete with any
+   * properties that can be provided at this time.
```

Comment:
[fix] The key thing to call out here is that the host should set any properties that it _owns_. Ie. any properties that are omitted here may well be `resolved` during the publishing process, before `register`. We should also note that managers may error if this is missing prerequisite data etc... (see the description on the issue for more on this).

Reply:
Might it be good to simplify the method docstrings and instead update the glossary section for `preflight`, then cross-reference it in `Manager` and `ManagerInterface`.  We need to define what "owns" means, along with he error modes.  Probably better to centralise that doc than burying it under the doc of a parameter.

Reply:
The acceptance criteria in the issue does simply say "Supporting documentation ... That a host should always provide known data up front." - but we're saying known isn't really the point, it's "owned"  that's important.

So if a host has previously queried some data for from the manager for the entity reference to be `preflight`ed, then there's no need to provide it to `preflight`, since that information is "owned" by the manager.  Makes sense.

Reply:
Shortened, used the word "owns", and linked to an expanded Glossary entry, in daa8a1ac